packages/manager/modules/pci/src/components/project/billing/resource-usage/translations/Messages_fr_FR.json

```diff
@@ -2,6 +2,9 @@
   "pci_billing_private_registry_no_entry": "Pas de consommation pour la période sélectionnée.",
   "pci_billing_private_registry_region": "Région",
   "pci_billing_private_registry_type": "Type",
+  "pci_billing_private_registry_type_cold_archive_archive": "Archive",
+  "pci_billing_private_registry_type_cold_archive_archive-fees": "Restauration",
+  "pci_billing_private_registry_type_cold_archive_penality-fees": "Pénalité",
```

Comment:
I dont see penalty_fees in constant file here packages/manager/modules/pci/src/components/project/billing/resource-usage/constants.js